torch/csrc/jit/tracing_state.h

```diff
@@ -150,6 +159,16 @@ struct TORCH_API NoWarn {
   bool prev;
 };
 
+struct WithTracingEnvStack {
+  WithTracingEnvStack() {
```

Comment:
The name doesn't clearly describe what it does. `WithNestedTracingFrame` might be better.